### PR TITLE
Add wallet model support for Dedicated ETH Staking

### PR DESCRIPTION
**Pull request by drohit-cb**
### What changed? Why?

This PR helps introduce dedicated eth staking for wallet mode.

### CODE REVIEW

**Review by melissamforbs — APPROVED** [review]

**Review by melissamforbs — APPROVED** [review]

File: src/coinbase/validator.ts [hunk 1]
```diff
@@ -90,6 +90,6 @@ export class Validator {
    * @returns The string representation of the Validator.
    */
   public toString(): string {
-    return `Id: ${this.getValidatorId()}, Status: ${this.getStatus()}`;
+    return `Id: ${this.getValidatorId()} Status: ${this.getStatus()}`;
```
Comment (drohit-cb) [inline src/coinbase/validator.ts:93]: the comma make it harder to copy/paster validator id which seems to be a common use-case after listing validators.

File: src/coinbase/staking_operation.ts [hunk 2]
```diff
@@ -202,4 +250,30 @@ export class StakingOperation {
       }
     }
   }
+
+  /**
+   * loadTransactionsFromModel loads new unsigned transactions from the model into the transactions array.
+   * Note: For External Address model since tx signing and broadcast status happens by the end user and not our backend
+   * we need to be careful to not overwrite the transactions array with the response from the API. Ex: End user could have used
+   * stakingOperation.sign() method to sign the transactions, and we should not overwrite them with the response from the API.
+   * This however is ok to do so for the Wallet Address model since the transactions are signed and broadcasted by our backend.
```
Comment (marcin-cb) [inline src/coinbase/staking_operation.ts:259]: To my knowledge the signing still happens on the client side but only broadcasting happens in the backend.

Comment (drohit-cb) [reply src/coinbase/staking_operation.ts:259]: Sry can you point which part of the above comment you disagree with ? it's a long one 😅

Comment (drohit-cb) [reply src/coinbase/staking_operation.ts]: Aah got it. Good catch. Reworded it completely. Take a look.

**Review by marcin-cb — APPROVED** [review]

**Pull request merged**